src/wrappers/themis/react-native-themis/src/index.d.ts

```diff
@@ -1,7 +1,7 @@
 export declare const COMPARATOR_NOT_READY: any, COMPARATOR_NOT_MATCH: any, COMPARATOR_MATCH: any, COMPARATOR_ERROR: any, KEYTYPE_RSA: any, KEYTYPE_EC: any;
 export declare function isBase64(str: String): boolean;
 export declare function string64(input: String): String;
-export declare function keyPair64(typeOfKey: any): Promise<Object>;
+export declare function keyPair64(typeOfKey?: any): Promise<Object>;
```

Comment:
Why is this optional? Definition seems to still require a proper value.

Reply:
`export function keyPair64(typeOfKey: any = KEYTYPE_EC): Promise<Object>`
Because the parameter typeOfKey has the default value. You can just call keyPair64() function without any parameters.